src/data/recipes.js

```diff
@@ -56,6 +56,20 @@ export const butter = itemify({
   recipeType: recipeType.KITCHEN,
 })
 
+/**
+ * @property farmhand.module:recipes.oliveOil
+ * @type {farmhand.recipe}
+ */
+export const oliveOil = itemify({
+  id: 'olive-oil',
+  name: 'Olive Oil',
+  ingredients: {
+    [items.olive.id]: 15,
```

Comment:
Apparently it takes [an incredible amount of olives to produce olive oil](https://oliveemotion.com/en/how-many-olives-are-needed-to-make-one-liter-of-olive-oil/):

> To obtain 1 liter of extra virgin olive oil, 5 to 6 kg of olive are required, approximately 5200 to 8000 olives.

That might be a little extreme for what we're going for, but what do you think about upping the challenge here to be slightly closer to reality? Maybe ~250 olives?

Reply:
oh wow, considering most recipes that use olive oil probably only need one i'm cool with upping it. it's going to be worth an insane amount based on that so we should probably also change the name to "extra virgin olive oil" haha. 

this also makes me wonder if the vegetable oil should have used more soybeans.... i've merged that but maybe we should alter it before releasing the update. thoughts?

Reply:
These both sounds like good ideas to me! #372 is merged but not yet released, so tweaking it should be no problem.

Reply:
looks like producing vegetable oil from soybeans is even more extreme haha. 7-8kg per liter is what i found, and an average bean weighs like .15g. granted a soybean plant produces more than just one bean, it's still dozens of thousands. our soybean crop does grow quickly though, so maybe like 300 or 400? 400 sounds like a lot but it's less than fully filling the largest field twice so maybe that's ok?

Reply:
i split the difference and went 350 soybeans for veggie oil; and went with your suggestion of 250 for olive oil

Reply:
That all seems balanced to me. Thanks for making the change!